### ISSUE TITLE
[Feedback] Conan is using old string formatting style

### ISSUE THREAD

**Issue by itai-huddly** [conan-io/training#41]
Currently enjoying the new conan course on JFrog Academy, so far I'm very happy with the content and the exercises.

One thing I've noticed is that some of the Python code generated by conan is using an older Python style for formatting strings.

So instead of 
`self.run(".{}example".format(os.sep))`

an older style is used:
`self.run(".%sexample" % os.sep)`

I would suggest updating the format to the newer style.

**Comment by memsharded** [comment]
Yes, good point. We should start adopting this style, not only in the docs, but everywhere, including the internal codebase.
